configs/pointpillars/hv_pointpillars_secfpn_6x8_160e_plus-kitti-3d-4class1.py

```diff
@@ -0,0 +1,117 @@
+_base_ = [
```

Comment:
what is difference between  `hv_pointpillars_secfpn_6x8_160e_plus-kitti-3d-4class.py`  and `hv_pointpillars_secfpn_6x8_160e_plus-kitti-3d-4class1.py`? 
if they are 2 different config, rename to a clear name; if same , remain 1 config.